packages/material-ui-lab/src/DesktopDatePicker/DesktopDatePicker.tsx

```diff
@@ -47,9 +47,14 @@ const DesktopDatePicker = React.forwardRef(function DesktopDatePicker<TDate>(
   const validationError = useDateValidation(props) !== null;
   const { pickerProps, inputProps, wrapperProps } = usePickerState(props, valueManager);
 
-  // Note that we are passing down all the value without spread.
-  // It saves us >1kb gzip and make any prop available automatically on any level down.
```

Comment:
> make any prop available automatically on any level down

That's not a good thing if we don't see what props are available where because they'll eventually get used creating a coupling that might've not been intended.